docs/_posts/2021-09-20-jekyll-4-2-1-released.markdown

```diff
@@ -0,0 +1,40 @@
+---
+title: "Jekyll 4.2.1 Released"
+date: 2021-09-20 12:45:46 +0530
+author: ashmaroli
+version: 4.2.1
+category: release
+---
+
+Hello Jekyllers!
+
+The Jekyll team is happy to announce the release of `v4.2.1` which fixes a couple of
+regressions introduced in `v4.2.0` and another bug inherited from Jekyll 3.
+
+In `v4.2.0`, we decided to stop overriding {% raw %}`{{ site.url }}`{% endraw %} with
+the *localhost* address when running the command `jekyll serve` with the default
+*development* mode. While the intent behind the change was to avoid forcing users to
+generate a *production build* separately by invoking `jekyll build`, it however had an
+unforeseen undesired consequence &mdash; absolute URLs for assets now pointed to
```

Comment:
```suggestion
unforeseen consequence &mdash; absolute URLs for assets now pointed to
```

having `unforeseen` and `undesired` together feels weird when reading this, so i took out `undesired` since the fact that we changed it made it obvious it was a undesired change. 😄